devops/Makefile

```diff
@@ -80,6 +79,14 @@ build-patroni:
 	@oc -n $(BUILD_NAMESPACE) process -f $(BUILD_TEMPLATE_PATH)  -p NAME=patroni -p TAG="12-latest" -p BASE_IMAGE_NAME=postgres -p BASE_IMAGE_TAG="12" -p SOURCE_REPOSITORY_URL="https://github.com/BCDevOps/platform-services.git" -p SOURCE_REPOSITORY_REF="master" -p SOURCE_CONTEXT_DIR="apps/pgsql/patroni/docker" | oc -n $(BUILD_NAMESPACE) apply -f -
 	@oc -n $(BUILD_NAMESPACE) start-build bc/patroni --wait
 
+init-api:
+	@echo "++\n Init Api in $(NAMESPACE) \n++"
+	test -n $(API_INIT_BCeID_WEB_SERVICE_ONLINE_SERVICE_ID)
+	test -n $(API_INIT_BCeID_WEB_SERVICE_REQUESTER_USER_GUID)
+	test -n $(API_INIT_BCeID_WEB_SERVICE_AUTH_USER_NAME)
+	test -n $(API_INIT_REQ_BCeID_WEB_SERVICE_AUTH_USER_PASSWORD)
+	@oc -n $(NAMESPACE) process -f openshift/api-pre-req.yml -p NAME=api -p BCeID_WEB_SERVICE_ONLINE_SERVICE_ID=$(API_INIT_BCeID_WEB_SERVICE_ONLINE_SERVICE_ID) -p BCeID_WEB_SERVICE_REQUESTER_USER_GUID=$(API_INIT_BCeID_WEB_SERVICE_REQUESTER_USER_GUID) -p BCeID_WEB_SERVICE_AUTH_USER_NAME=$(API_INIT_BCeID_WEB_SERVICE_AUTH_USER_NAME) -p BCeID_WEB_SERVICE_AUTH_USER_PASSWORD=$(API_INIT_REQ_BCeID_WEB_SERVICE_AUTH_USER_PASSWORD) | oc -n $(NAMESPACE) apply -f -
```

Comment:
Do we need to provide the oc -n ${namespace} twice? Once at the start once at the end?

Reply:
To invoke the make command it will be provided only once (e.g. "make init-api NAMESPACE=<some_namespace>").
Here we have two OC commands being executed (process/apply) and both are providing the NAMESPACE parameter, following the same that others in the same makefile are doing.